terraform/groups/ecs-service/main.tf

```diff
@@ -19,7 +19,7 @@ terraform {
 }
 
```

Comment:
why is this file here?

Reply:
I had to rebase against the master branch because Seun made a change. This file was introduced during the rebase.

Reply:
your rebase is wrong, your changes should not show other peoples commitnwhen rebased properly.
Have shown you how to rebase many time. Please correct this,

Reply:
I tried to fix the rebase, but was unable to. Asked Maciej for help, but we both surmised the only option was to create a new branch and port the changes over. This is done now. New PR: https://github.com/companieshouse/accounts-association-api/pull/42